Projects/Modules/NetworkModule/Sources/Error/NetworkError.swift

```diff
@@ -1,16 +1,20 @@
 import Foundation
 
-enum RequestError: Error, LocalizedError {
-    case missingURL
+enum NetworkError: Error, LocalizedError {
+    case invaildURL
     case jsonEncodingFailed(Error)
+    case invaildResponse
     
     var errorDescription: String? {
         switch self {
-        case .missingURL:
-            return "요청에서 유효한 URL을 찾을 수 없습니다."
+        case .invaildURL:
+            return "유효한 URL을 찾을 수 없습니다."
```

Comment:
`return`을 생략할지 컨벤션 룰을 얘기해보면 좋을 것 같내요.

```suggestion
        case .invaildURL:
           "유효한 URL을 찾을 수 없습니다."
```